model/derived/gasoutputs.go

```diff
@@ -65,8 +67,3 @@ func (l GasOutputsList) PersistWithTx(ctx context.Context, tx *pg.Tx) error {
 	}
 	return nil
 }
-
-type ProcessingGasOutputs struct {
-	Height int64
-	GasOutputs
-}
```

Comment:
I am not sure why this was added this way, but if we include height in the output view, then we don't need it.

Reply:
I think I added it because I realised height was missing from the underlying table but I needed to return it here to optimize the query